kube-derive/src/lib.rs

```diff
@@ -111,6 +111,22 @@ mod custom_resource;
 /// Adding `#[kube(derive = "PartialEq")]` is required if you want your generated
 /// top level type to be able to `#[derive(PartialEq)]`
 ///
+/// ### `#[kube(schema = "mode")]`
+/// Defines whether the `JsonSchema` of the top level generated type should be used when generating a `CustomResourceDefinition`.
+///
+/// Legal values:
+/// - `"derived"`: A `JsonSchema` implementation is automatically derived
+/// - `"custom"`: `JsonSchema` is not derived, but used when creating the `CustomResourceDefinition` object
```

Comment:
Maybe "manual"? "custom" is fine, but we already have lots of "custom" (`CustomResource`, `CustomResourceDefinition`, "custom schema", "customize"), and I think it works well with "derived".

Reply:
42d8170